causal_testing/testing/estimators.py

```diff
@@ -71,7 +72,7 @@ class LinearRegressionEstimator(Estimator):
     combination of parameters and functions of the variables (note these functions need not be linear).
     """
     def __init__(self, treatment: tuple, treatment_values: float, control_values: float, adjustment_set: set,
-                 outcome: tuple, df: pd.DataFrame = None, effect_modifiers: {Variable: any} = None, product_terms: [(Variable, Variable)] = None):
+                 outcome: tuple, df: pd.DataFrame = None, effect_modifiers: set[Variable: Any] = None, product_terms: list[tuple[Variable, Variable]] = None):
```

Comment:
dict

Reply:
Thanks for spotting @jmafoster1 :), corrected them all now